src/main/java/com/google/api/codegen/py/PythonSphinxCommentFixer.java

```diff
@@ -22,6 +22,8 @@
 
   /** Returns a Sphinx-formatted comment string. */
   public static String sphinxify(String comment) {
+    // escape '$' in the comment first
+    comment = comment.replaceAll("\\$", "\\\\\\$");
```

Comment:
Do we need to undo the extra backslashes at the end so that the generated comment doesn't contain the escape characters?